recipe/meta.yaml

```diff
@@ -1,5 +1,6 @@
 {% set name = "aio-pika" %}
 {% set version = "9.5.5" %}
+{% set python_min = "3.9" %}
```

Comment:
Thanks for this PR.  Can you describe the need for this line?

Reply:
This is encouraged by CFEP-25 https://github.com/conda-forge/cfep/blob/main/cfep-25.md

Reply:
If you need to differ from what's set by conda-forge, I'd recommend removing it and rerendering.